src/Workleap.Extensions.Mongo.Ephemeral/ConfigureEphemeralMongoClientOptions.cs

```diff
@@ -13,11 +13,11 @@ public ConfigureEphemeralMongoClientOptions(DefaultDatabaseNameHolder defaultDat
         this._runnerProvider = runnerProvider;
     }
 
-    public void Configure(string name, MongoClientOptions options)
+    public void Configure(string? name, MongoClientOptions options)
     {
         // Each test that requests a IMongoDatabase will have its own database
         // There will also be one MongoDB instance per named MongoDB client
-        var runner = this._runnerProvider.GetRunner(name);
+        var runner = this._runnerProvider.GetRunner(name ?? "");
```

Comment:
nit: use Options.DefaultName instead of ""

Reply:
Good point!